### PR TITLE
Translated project-gateway.md.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/project-administration/project-gateway.md [hunk 1]
```diff
@@ -1,72 +1,72 @@
...
 ## NodePort
 
-If you select **NodePort**, KubeSphere will set a port for http and https requests respectively. You can access your Service at `EIP:NodePort` or `Hostname:NodePort`.
+如果您选择 **NodePort**，KubeSphere 将为 HTTP 请求和 HTTPS 请求分别设置一个端口。您可以用 `EIP:NodePort` 或 `Hostname:NodePort` 地址访问服务。
 
-![nodeport](/images/docs/project-administration/project-gateway/nodeport.jpg)
+![nodeport](/images/docs/zh-cn/project-administration/project-gateway/nodeport.jpg)
 
-For example, to access your Service with an elastic IP address (EIP), visit:
+例如，如果您的服务配置了的弹性IP地址 (EIP)，请访问：
```
Comment (Sherlock113) [inline content/zh/docs/project-administration/project-gateway.md:43]: 弹性IP地址
->
弹性 IP 地址

File: content/zh/docs/project-administration/project-gateway.md [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
 - `https://demo.kubesphere.io:32471`
 
 {{< notice note >}}
 
-- You may need to open ports in your security groups and configure relevant port forwarding rules depending on your environment.
+- 取决于您的环境，您可能需要在安全组中放行端口并配置相关的端口转发规则 。
 
-- If you access your Service using the host name, make sure the domain name you set can be resolved to the IP address.
-- **NodePort** is not recommended for a production environment. You can use **LoadBalancer** instead.
+- 如果使用主机名访问服务，请确保您设置的域名可以解析为对应的IP地址。
```
Comment (Sherlock113) [inline content/zh/docs/project-administration/project-gateway.md:57]: 对应的IP地址
->
对应的 IP 地址

File: content/zh/docs/project-administration/project-gateway.md [hunk 3]
```diff
@@ -1,72 +1,72 @@
...
-You must configure a load balancer in advance before you select **LoadBalancer**. The IP address of the load balancer will be bound to the gateway to provide access to internal Services and Routes. 
+在选择 **LoadBalancer** 前，您必须先配置负载均衡器。负载均衡器的 IP 地址将与网关绑定以便内部的服务和路由可以访问。 
 
-![lb](/images/docs/project-administration/project-gateway/lb.png)
+![lb](/images/docs/zh-cn/project-administration/project-gateway/lb.png)
 
 {{< notice note >}}
 
-Cloud providers often support load balancer plugins. If you install KubeSphere on major Kubernetes engines on their platforms, you may notice a load balancer is already available in the environment for you to use. If you install KubeSphere in a bare metal environment, you can use [Porter](https://github.com/kubesphere/porter) for load balancing.
+云厂商通常支持负载均衡器插件。如果在平台上主流的 Kubernetes Engine 上安装 KubeSphere，您可能会发现环境中已有可用的负载均衡器。如果在裸金属环境中安装 KubeSphere，您可以使用 [Porter](https://github.com/kubesphere/porter) 作为负载均衡器。
```
Comment (Sherlock113) [inline content/zh/docs/project-administration/project-gateway.md:70]: 如果在平台上主流的 Kubernetes Engine 上
->
如果在主流的 Kubernetes Engine 上

Delete "on their platforms" in the English source text.

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**